src/serve-static.ts

```diff
@@ -14,36 +13,85 @@ export type ServeStaticOptions = {
   rewriteRequestPath?: (path: string) => string
 }
 
-export const serveStatic = (options: ServeStaticOptions = { root: '' }) => {
-  return async (c: Context, next: Next): Promise<Response | undefined> => {
+const createStreamBody = (stream: ReadStream) => {
```

Comment:
Here we can use `Readable.toWeb(ReadStream)` instead, thus we can fully support backpressure.

Reply:
Okay. Now let us you `Readable.toWeb()`. Thanks!